src/Blaster.WebApi/Features/Shared/userservice.js

```diff
@@ -1,9 +1,111 @@
-const currentUser = window.currentUser;
-
-export { currentUser as currentUser }
+import * as Msal from "msal";
 
 export default class UserService {
+    constructor(configuration)
+    {
+        if (!configuration) {
+            configuration = {
+                auth: {
+                    clientId: "91c38c20-4d2c-485d-80ac-a053619a02db",
+                    authority: "https://login.microsoftonline.com/common",
+                    redirectUri: "http://localhost:5000"

```

Comment:
Take care of this.

Reply:
Removed redirect uri, which means default behavior will be using a popup, not a redirect. :)